### PR TITLE
KOGITO-7118: Establish marshaller roundtrip test runner for sw-editor-kogito-app

### PR DESCRIPTION
**Pull request by domhanak**
Create a integration test profile for sw-editor-kogito-app.
Create a HelloWorldExample test.
Implement skeleton for marshaller roundtrip of sw editor.

Employs same concept as we use in `stunner-editors` only difference we use [JsonUnit](https://github.com/lukas-krecan/JsonUnit)
which is a superb library for asserting json, also integrated with assertj <3

### CODE REVIEW

**Comment by domhanak** [comment]
Forgot to fix the dependency version management, working on it... :)

**Comment by domhanak** [comment]
NOTE: As of now the test will fail as the `getContent()` returns non-equal JSON because it adds an extra property.

You can run the tests using `mvn verify -Psw-editor-it-tests`, of course the app needs to be compiled.

File: packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-kogito-app/src/test/java/org/kie/workbench/common/stunner/sw/client/selenium/SWEditorSeleniumIT.java [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+            throw new IllegalStateException("The screenshotDir must be writable" + scd);
+        }
+        return scd;
+    }
+
+    private void waitCanvasPanel() {
+        final WebElement canvasPanelDiv = waitOperation()
+                .until(visibilityOfElementLocated(className(CANVAS_PANEL)));
+        assertThat(canvasPanelDiv)
+                .as("Once content is set canvas panel isibility is a prerequisite" +
```
Comment (tomasdavidorg) [inline packages/serverless-workflow-diagram-editor/sw-editor/sw-editor-kogito-app/src/test/java/org/kie/workbench/common/stunner/sw/client/selenium/SWEditorSeleniumIT.java]:
```suggestion
                .as("Once content is set, canvas panel visibility is a prerequisite" +
```

**Review by tomasdavidorg — APPROVED** [review]
Looks good to me 🙂, just small typo.

**Review by romartin — APPROVED** [review]

**Comment by domhanak** [comment]
@tiagobento safe to merge, as the profile is not triggered in build so nothing will break here, we will include it in PR checks once we figure out some small issues with current marshalling like - https://issues.redhat.com/browse/KOGITO-7240

**Comment by domhanak** [comment]
@tiagobento please merge :angel:

**Pull request merged**